src/main/java/touro/ConwaysGameOfLife/Grid.java

```diff
@@ -3,36 +3,36 @@
 public class Grid {
 
     private boolean[][] board;
-    private  Direction[] directions = Direction.values();
+    private Direction[] directions = Direction.values();
 
-    private final int HEIGHT = 40;
     private final int WIDTH = 40;
+    private final int HEIGHT = 40;
 
-    private boolean isAlive = true;
 
     public Grid() {
-        this.board = new boolean[HEIGHT][WIDTH];
+        this.board = new boolean[WIDTH][HEIGHT];
     }
 
 
     public void makeMove() {
-        boolean[][] tempBoard = new boolean[HEIGHT][WIDTH];
+        boolean[][] tempBoard = new boolean[WIDTH][HEIGHT];
 
-        for (int y = 0; y < HEIGHT; y++) {
-            for (int x = 0; x < WIDTH; x++) {
+        for (int x = 0; x < WIDTH; x++) {
+            for (int y = 0; y < HEIGHT; y++) {
                 int aliveNeighbors = countAliveNeighbors(x, y);
 
-                if (isAlive(x, y)) {
-                    if (aliveNeighbors < 2) {
-                        setDead(x,y);
+                if (board[x][y]) {
+                    if (aliveNeighbors < 2 || aliveNeighbors > 3) {
+                        board[x][y] = false;
+                        //setDead(x, y);
```

Comment:
we could get rid of these comments and methods if they are doing the same thing as setting board[x][y] to true or false.